src/platform/plugins/shared/data_view_management/public/components/field_editor/components/scripting_call_outs/disabled_call_out.tsx

```diff
@@ -17,6 +17,7 @@ export const ScriptingDisabledCallOut = ({ isVisible = false }) => {
   return isVisible ? (
     <Fragment>
       <EuiCallOut
+        announceOnMount
```

Comment:
```suggestion
        announceOnMount={false}
```
Also appears static.